numpy/testing/overrides.py

```diff
@@ -0,0 +1,44 @@
+"""Tools for testing implementations of __array_function__ and ufunc overrides
+
+
+"""
+
+from numpy.core.overrides import ARRAY_FUNCTIONS as array_functions
+from numpy import ufunc as _ufunc
+import numpy.core.umath as _umath
+
+ufuncs = {obj for obj in _umath.__dict__.values() if isinstance(obj, _ufunc)}
```

Comment:
I didn't know there were ufuncs in there! This set should only include things in the public numpy API, so no I don't think they should be included. I've also added a note to the docs that `ufuncs` and `array_functions` only include things in the public numpy API.

Reply:
Right, they are probably wrapped by `__array_function__` anyway (in functions that support it).  One day it would be nice to figure out the part of falling back to NumPy for part of the array-function API (it doesn't seem impossible, but needs someone dedicated to do some careful scouting, I suspect).
Only then it would probably even start to make sense to bless some ufuncs to explicitly show up here (e.g. the internal clip ufunc).